ddtrace/span.py

```diff
@@ -283,8 +283,12 @@ def get_metric(self, key):
         return self.metrics.get(key)
 
     def to_dict(self):
+        trace_id = self.trace_id
+        # ensure 128 bit trace IDs are always trimmed
```

Comment:
Is this comment accurate? It seems like any `trace_id` longer than 63 bits will be trimmed.

Reply:
bit_length returns `63` for the max 64bit integer